src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -15,30 +18,49 @@ namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
     /// </summary>
     internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
     {
+        public const string TagId = "inline parameter name hints";
+
         /// <summary>
         /// Creates the UIElement on call
         /// Uses PositionAffinity.Successor because we want the tag to be associated with the following character
         /// </summary>
         /// <param name="text">The name of the parameter associated with the argument</param>
-        public InlineParameterNameHintsTag(string text)
-            : base(CreateElement(text), removalCallback: null, PositionAffinity.Successor)
+        public InlineParameterNameHintsTag(string text, double lineHeight, TextFormattingRunProperties format)
+            : base(CreateElement(text, lineHeight, format), removalCallback: null, PositionAffinity.Successor)
         {
         }
 
-        private static UIElement CreateElement(string text)
+        private static UIElement CreateElement(string text, double lineHeight, TextFormattingRunProperties format)
         {
-            // Constructs the hint block which gets assigned parameter name, a normal fontstyle, and sets the padding 
-            // space around the block to 0
+            // Constructs the hint block which gets assigned parameter name and nested inside a border where the 
+            // attributes of the block are received from the options menu
             var block = new TextBlock
             {
-                Text = text + ": ",
+                Text = text + ":",
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
                 FontStyle = FontStyles.Normal,
+                FontFamily = format.Typeface.FontFamily,
+                FontSize = format.FontRenderingEmSize - (0.25 * format.FontHintingEmSize),
```

Comment:
doc.  also, can you post some pictures at different font sizes in the editor, different zoom sizes in teh editor, and different DPI levels for windows?

Reply:
1. 
Font Size 7:
![image](https://user-images.githubusercontent.com/40616383/87189489-bbe2d000-c2be-11ea-9abf-a00acd4d7396.png)

Font Size 14:
![image](https://user-images.githubusercontent.com/40616383/87189512-cdc47300-c2be-11ea-8094-57cfa4b631e8.png)

Font Size 19:
![image](https://user-images.githubusercontent.com/40616383/87189425-9e156b00-c2be-11ea-9568-12fc34484ce8.png)

Font Size 24:
![image](https://user-images.githubusercontent.com/40616383/87189582-e6cd2400-c2be-11ea-9384-e8aa21043e98.png)

Reply:
2.
Comic Sans:
![image](https://user-images.githubusercontent.com/40616383/87189641-019f9880-c2bf-11ea-8bf3-b3723f2f0a9a.png)

Elephant:
![image](https://user-images.githubusercontent.com/40616383/87189686-11b77800-c2bf-11ea-8d6e-3d7ae7ba0c53.png)

Pristina:
![image](https://user-images.githubusercontent.com/40616383/87189764-39a6db80-c2bf-11ea-8799-34c3d1c08cfb.png)

Reply:
3.

50%:
![image](https://user-images.githubusercontent.com/40616383/87189840-62c76c00-c2bf-11ea-9606-136463508cfe.png)

67%:
![image](https://user-images.githubusercontent.com/40616383/87189880-7541a580-c2bf-11ea-9a4c-7ee7426788c8.png)

100%:
![image](https://user-images.githubusercontent.com/40616383/87189913-838fc180-c2bf-11ea-9c2a-547e2f55dd99.png)

137%:
![image](https://user-images.githubusercontent.com/40616383/87189962-97d3be80-c2bf-11ea-91ed-d94e6cf64a97.png)

Reply:
4.

DPI Level 100%:
![image](https://user-images.githubusercontent.com/40616383/87190322-424be180-c2c0-11ea-8756-914df355c5db.png)

DPI Level 125%:
![image](https://user-images.githubusercontent.com/40616383/87190361-52fc5780-c2c0-11ea-971e-2769a59ed06a.png)

DPI Level 150% (Recommended for my computer):
![image](https://user-images.githubusercontent.com/40616383/87190414-6c9d9f00-c2c0-11ea-8c0b-13620439175f.png)

DPI Level 175%:
![image](https://user-images.githubusercontent.com/40616383/87190243-234d4f80-c2c0-11ea-8d5b-1e3ab81d69a2.png)